packages/sdk/README.md

```diff
@@ -156,6 +156,187 @@ channel.banUser(message.sender);
 
 To verify if the user was banned, check the property `isBanned` in the user object.
 
+### Q&A
+
+If the "Q&A" feature is enabled for your chat plan and activate for the current chat, you can implement it using this module. 
+
+First you'll need to get the `qnaProps` with your `chatId` 
+
+```typescript
+import Qna from '@/qna/qna.ts'
+
+// get qna props
+const qnaProps: QnaProps = await Qna.getQnaProps(chatId)
+```
+
+With props in hands, pass it to Qna constructor along with the `qnaId`, the `site` wich is defined in [types](https://github.com/stationfy/arena-chat-sdk/tree/feature/sprint95-update-chat-sdk-doc/packages/types) and the previous defined `arenaChat`
+
+```typescript
+// get instance of Qna
+const qna = new Qna(qnaProps, qnaId, site, arenaChat)
+```
+
+Now is possible to start loading the previously created questions. Just pass a limit of questions to be loaded and the QnaQuestionFilter, wich is also defined in [types](https://github.com/stationfy/arena-chat-sdk/tree/feature/sprint95-update-chat-sdk-doc/packages/types). Both parameters are optional
+
+```typescript
+const questions: [QnaQuestion] = await qna.loadQuestions(50, QnaQuestionFilter.RECENT)
+```
+
+Start adding new questions just by passing its contents
+
+```typescript
+await qna.addQuestion("Which team shall win tonight?")
+```
+
+It's also possible to easily awnser a question by calling the following method
+
+```typescript
+const isQuestionAwnsered: Boolean = await qna.awnserQuestion(questions[0], "Lakers should win!")
+```
+
+Now there are some event listeners that should be noticed
+
+- onChange: This will watch for Q&A props changes coming from dashboard and then call the passed callback with the Qna instance updating it properties
+```typescript
+onChange(callback: (instance: BaseQna) => void): void
+```
+
+- onQuestionReceived: Watchs for new questions updating the question cache and calls the passed callback with the new question
+```typescript
+onQuestionReceived(callback: (question: QnaQuestion) => void): void
+```
+
+### Direct Messages 
+
+By having the "Private Messages" option enabled in your chat, it's possible to create private dialogs between two or more users.
```

Comment:
When you enable the "Private Messages" option in you chat, members can chat directly with each other.